Classes/NodeEnumeration/NodeEnumerator.php

```diff
@@ -50,8 +50,7 @@ class NodeEnumerator
      */
     protected $nodeTypeWhitelist;
 
-
-    public function enumerateAndStoreInRedis(?Site $site, ContentReleaseLogger $contentReleaseLogger, ContentReleaseIdentifier $releaseIdentifier)
+    public function enumerateAndStoreInRedis(?Site $site, ContentReleaseLogger $contentReleaseLogger, ContentReleaseIdentifier $releaseIdentifier): void
```

Comment:
I'd suggest we *add* the workspace name here, and make it configrable through the CLI command. (and use this one; instead of the $newMetadata->getWorkspaceName() below)